doc/reference/pipeline_spec.md

```diff
@@ -173,99 +175,106 @@ In practice, you rarely need to specify all the fields.  Most fields either come
 }
 ```
 
-Following is a walk-through of all the fields.
-
 ### Name (required)
 
-`pipeline.name` is the name of the pipeline that you are creating.  Each
-pipeline needs to have a unique name. Pipeline names must:
+`pipeline.name` is the name of the pipeline that you are creating. Each
+pipeline needs to have a unique name. Pipeline names must meet the following
+prerequisites:
 
-- contain only alphanumeric characters, `_` and `-`
-- begin or end with only alphanumeric characters (not `_` or `-`)
-- be no more than 50 characters in length
+- Include only alphanumeric characters, `_` and `-`.
+- Begin or end with only alphanumeric characters (not `_` or `-`).
+- Do not exceed 50 characters in length.
```

Comment:
bullet list should have the same structure:

Pipeline names must... include only...
Pipeline name must..begin or end with...
Pipeline name must... do not exceed... <-- doesn't match. 

remove the "do"?